### ISSUE TITLE
`WOTKernel.compute_transition_matrix` does not fail with random argument

### ISSUE THREAD

**Issue by WeilerP** [theislab/cellrank#721]
## Description

Passing a random argument to `WOTKernel.compute_transition_matrix` does not throw an error. In this case, the `TypeError`

```pytb
TypeError: compute_transition_matrix() got an unexpected keyword argument 'random_kwarg'
```

should be thrown, IMO.

### Code snippet

```python
adata = cr.datasets.reprogramming_schiebinger()
adata = adata[adata.obs["serum"] == "True"].copy()
sc.pp.subsample(adata, fraction=0.25, random_state=0)
adata

sc.pp.pca(adata)
sc.pp.neighbors(adata, random_state=0)

wk = WOTKernel(adata, time_key="day")

wk.compute_initial_growth_rates(organism="mouse", key_added="growth_rate_init")
wk.compute_transition_matrix(
    growth_iters=3,
    growth_rate_key="growth_rate_init",
    last_time_point="connectivities",
    random_kwarg=1
)
```

### Versions

This is happening both for `cellrank==1.4.0` and on `cellrank@dev`.

**Comment by michalk8** [comment]
IWe can pass `kwargs` to the OT configuration - it is unfortunately beyond my control whether they silently throw invalid ones away or not. The best we can do is to look if the keys are present in [here](https://github.com/broadinstitute/wot/blob/master/wot/ot/ot_model.py#L85) and if not, raise `TypeError`, but it really shouldn't be our problem to deal with.

**Comment by WeilerP** [comment]
Okay, I can open an issue on the WOT repo.

**Comment by michalk8** [comment]
> Okay, I can open an issue on the WOT repo.

I'd say either we fix it ourselves (easier). leave it be (easiest) or open an issue as you've suggested (will take time to fix, I think) - I leave the decision up to you @WeilerP .

**Comment by WeilerP** [comment]
@michalk8, I'll open an issue on the WOT repo. Just to double check: The error should (or is expected to ) be thrown by `wot.ot.OTModel`, right?

**Comment by michalk8** [comment]
Do you want to keep this issue open?

**Comment by WeilerP** [comment]
> Do you want to keep this issue open?

I'd say we have to fix it ourselves if we want it implemented. There doesn't seem to be much activity on the WOT repo. Haven't heard back since I opened the issue.

**Comment by michalk8** [comment]
closed via #737
